db/database_test.go

```diff
@@ -28,14 +28,9 @@ import (
 	"github.com/couchbase/sync_gateway/auth"
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/channels"
-	"github.com/robertkrimen/otto/underscore"
 	"github.com/stretchr/testify/assert"
 )
 
-func init() {
-	underscore.Disable() // It really slows down unit tests (by making otto.New take a lot longer)
-}
```

Comment:
we are still running some of these tests with otto - that said, I'd be fine with putting this into a separate PR and moving this to `main_test.go` for relevant tests - and we can remove when we remove otto.

Reply:
I took that line out because it doesn't make a significant difference anymore, i.e. the tests don't really take longer with it removed. I remember adding that back in 2013 or so, and either MacBooks have gotten a lot faster or the Go runtime has (probably more of the latter.)